.gitignore

```diff
@@ -1,8 +1,23 @@
-*.egg-info
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
+*.egg-info/
+*.profraw
 *.pyc
+*.pyo
 .coverage
+.installed.cfg
+.mr.developer.cfg
 .tox/
+__pycache__/
+bin/
+build/
 coverage.xml
+develop-eggs/
+dist/
 docs/_build
+eggs/
 htmlcov/
-.coverage.*
```

Comment:
Huh, so the meta template doesn't ignore .coverage.$pid, that gets produced by coverage run --parallel?  Maybe it should.